test/index.spec.tsx

```diff
@@ -102,15 +119,19 @@ describe('[Controlled, UnControlled]: init', () => {
 describe('[Controlled, UnControlled]: editorDidConfigure', () => {
 
   it('editorDidConfigure(editor)', () => {
+    let uCallback;
+    let uConfigured;
+    let cCallback;
+    let cConfigured;
 
     Enzyme.shallow(
       <UnControlled
         editorDidMount={(editor, value, next) => {
-          this.uCallback = sinon.spy(next);
-          this.uCallback();
+          uCallback = sinon.spy(next);
+          uCallback();
```

Comment:
TypeScript doesn't like this floating `this` (which ends up referring to `window`, which is how the tests previously passed).